doc/WebRTC.xml

```diff
@@ -0,0 +1,1022 @@
+                    <entry>
+                      <para>An optional identifier chosen by the client that can be used to
+                        correlate the response from the signaling server with the request. Echoed
+                        back in the signaling server response.</para>
+                    </entry>
+                  </row>
+                </tbody>
+              </tgroup>
+            </informaltable>
+          </para>
+        </section>
+        <section xml:id="section_lsq_qhn_rwb">
+          <title>Device-facing header keys</title>
+          <para>The following key/value pairs are used in the header part in device messages.</para>
+          <informaltable>
+            <tgroup cols="2">
+              <colspec colnum="1" colname="c1" colwidth="1*"/>
+              <colspec colnum="2" colname="c2" colwidth="3.55*"/>
+              <tbody>
+                <row>
+                  <entry>
+                    <para><emphasis role="bold">Key</emphasis></para>
+                  </entry>
+                  <entry>
+                    <para><emphasis role="bold">Description</emphasis></para>
+                  </entry>
+                </row>
+                <row>
+                  <entry>clientId</entry>
+                  <entry>The Signaling ID of the client to which the message is intended, or the
+                    client that sent the message. This ID is unique within the organization and the
+                    device gets this id from server during initSession.</entry>
+                </row>
+              </tbody>
+            </tgroup>
+          </informaltable>
+        </section>
+      </section>
+      <section xml:id="section_ozx_512_v5b">
+        <title>Payload keys</title>
+        <para>The following key/value pairs are used in the payload part of messages.</para>
+        <para>
+          <informaltable>
+            <tgroup cols="2">
+              <colspec colnum="1" colname="c1" colwidth="1*"/>
+              <colspec colnum="2" colname="c2" colwidth="3.76*"/>
+              <tbody>
+                <row>
+                  <entry>
+                    <para><emphasis role="bold">Key</emphasis></para>
+                  </entry>
+                  <entry>
+                    <para><emphasis role="bold">Description</emphasis></para>
+                  </entry>
+                </row>
+                <row>
+                  <entry>
+                    <para>apiVersion</para>
+                  </entry>
+                  <entry>
+                    <para>The API version that should be used in the request.</para>
+                  </entry>
+                </row>
+                <row>
+                  <entry>
+                    <para>type</para>
+                  </entry>
+                  <entry>Specifies whether this message is a request or response.</entry>
+                </row>
+                <row>
+                  <entry>
+                    <para>sessionId</para>
```

Comment:
The session ID being outside of the header is problematic because it prevents the field being used to correlate an entire session.
The principal usecase where this shows up is when doing an `initSession` for an offline camera. The signaling server must answer an `error` saying the device is offline, but it doesn't know the sessionId
This is problematic if the client is a bridge of some sort, that might be handling dozens or hundreds of sessions concurrently.